easybuild/easyconfigs/p/PeakSeq/PeakSeq-1.3-goolf-1.4.10.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = "PeakSeq"
+version = "1.3"
+
+homepage = 'http://info.gersteinlab.org/PeakSeq'
+description = """ PeakSeq is a program for identifying and ranking peak regions in ChIP-Seq 
+ experiments. It takes as input, mapped reads from a ChIP-Seq experiment, mapped reads from 
+ a control experiment and outputs a file with peak regions ranked with increasing Q-values."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://archive.gersteinlab.org/proj/PeakSeq/Scoring_ChIPSeq/Code/C/']
+sources = ['%(name)s.zip']
```

Comment:
since naming here of the package is broken anyway I recommend, if you agree, to make it:
`sources = [SOURCE_ZIP]` # ie. imply a manual renaming to correct version.
it may appear as a hassle in the beginning, but it surely pays back well down the road.